jobs/cloud/scheduler.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+package cloud
+
+import (
+	"time"
+
+	"github.com/mattermost/mattermost-server/v5/app"
+	"github.com/mattermost/mattermost-server/v5/model"
+)
+
+const (
+	SchedFreqMinutes = 2 // 1 day
```

Comment:
Is the comment accurate or we can get rid of it

Reply:
It will be accurate, it's like this for testing. Will leave this unresolved so I don't forget to move it back. Trying to figure out how to get this going in a spinwick for Steve and Matt.

Reply:
https://github.com/mattermost/enterprise/pull/821/commits/e1fe5f96fd1edd9d14dab6142e8cd846f3b331fa set to 1440, which is 1 day in minutes.

Reply:
Swapped to 720 minutes, so that it is guaranteed to happen once a day